src/views/actions/thunks.ts

```diff
@@ -127,7 +127,7 @@ export const getViewAndResultsForVEO = (
       })
     )
 
-    const queries = view.properties.queries.filter(({text}) => !!text.trim())
+    const queries = view.properties.queries.filter(({text}) => !!text)
```

Comment:
PUT IT BACK

Reply:
if i put it back, the query builder no longer caches!

Reply:
I think the real question is _why_ this is the fix tho. LMK if you can take a look w me at some point

Reply:
```js
!!' ' // true
!!' '.trim() // false
```